modules/sharing-registry/sharing-registry-server/src/main/resources/sharing-registry-mysql.sql

```diff
@@ -56,6 +56,21 @@ CREATE TABLE USER_GROUP (
   FOREIGN KEY (OWNER_ID, DOMAIN_ID) REFERENCES SHARING_USER(USER_ID, DOMAIN_ID) ON DELETE CASCADE ON UPDATE NO ACTION
 )ENGINE=InnoDB DEFAULT CHARACTER SET=latin1;
 
+CREATE TABLE GROUP_OWNER (
+  OWNER_ID VARCHAR(255) NOT NULL,
+  GROUP_ID VARCHAR(255) NOT NULL,
+  DOMAIN_ID VARCHAR(255) NOT NULL,
+  PRIMARY KEY (OWNER_ID, DOMAIN_ID),
```

Comment:
This primary key restricts users from owning more than one group. Perhaps the primary key should be `(GROUP_ID, DOMAIN_ID)`, which would restrict groups from having more than one owner?

Related question: USER_GROUP has an OWNER_ID column.  Do we need a GROUP_OWNER table?

Reply:
@sachinkariyattin i merged this PR before I reviewed @machristie comment above. After reviewing it back again, I fully agree with this above comments. Since groups can have multiple admins, it makes sense to have a GROUP_ADMIN table. But since groups can only have one owner, it makes sense to get rid of GROUP_OWNER table and use the OWNER_ID in the USER_GROUP table. Can you please make changes accordingly?

Reply:
Yes, didn't think about this. 
I will make these changes. Should I open a new PR?

Reply:
yes please, can you please create a new PR and reference this PR? -- thanks.

Reply:
I have made the changes and linked the PR